snapshot/windows/windows.go

```diff
@@ -4,16 +4,33 @@ package windows
 
 import (
 	"context"
+	"os"
+	"path/filepath"
+	"strings"
+	"syscall"
+	"unsafe"
 
+	"golang.org/x/sys/windows"
+
+	"github.com/Microsoft/hcsshim"
+	"github.com/containerd/containerd/errdefs"
+	"github.com/containerd/containerd/fs"
+	"github.com/containerd/containerd/log"
 	"github.com/containerd/containerd/mount"
 	"github.com/containerd/containerd/plugin"
 	"github.com/containerd/containerd/snapshot"
+	"github.com/containerd/containerd/snapshot/storage"
 	"github.com/pkg/errors"
 )
 
-var (
-	ErrNotImplemented = errors.New("not implemented")
-)
+type snapshotter struct {
+	root string
+	info hcsshim.DriverInfo
+	ms   *storage.MetaStore
+}
+
+// filterDriver is an HCSShim driver type for the Windows Filter driver.
+const filterDriver = 1
```

Comment:
Is this constant can be defined in `hccshim` package?

Reply:
If not, can we have it exported to use outside of this package (I've left a comment a bit above asking to add a constant)

Reply:
Yes, this will be a constant in hcsshim package and not hard coded.